sync/pipelines.yaml

```diff
@@ -0,0 +1,68 @@
+# Each YAML file under sync/ configures how helper/helper.py synchronizes
+# contents of various versions from its source of truth (usually a GitHub
+# repository of a Tekton component, such as tektoncd/pipelines) to
+# content/ (for the lastest version) and vault/ (for earlier versions).
+
+# The name of the component.
+# helper.py will use this value to build directories in content/ and vault/.
+component: Pipelines
+# The order of the component.
+displayOrder: 0
+# The GitHub repository where documentation resides.
+repository: https://github.com/tektoncd/pipeline
+# The directory in the GitHub repository where contents reside.
+docDirectory: docs
+# The tags (versions) of contents to sync.
+# Note that helper.py and related script reads tags in the order specified in
+# the following list; the first entry in tags will automatically become the
+# latest version of contents.
+tags:
+  # The name of the tag in the GitHub repository.
+- name: v0.9.1
+  # The name to display on tekton.dev.
+  # helper.py will use this value in the version switcher and other places.
+  displayName: v0.9.x
+  # Key-value pairs of files to sync, where the key is the original filename
+  # and the value is the new filename.
+  files:
+  - README.md: _index.md
+  - auth.md: auth.md
+  - conditions.md: conditions.md
+  - container-contract.md: container-contract.md
+  - labels.md: labels.md
+  - logs.md: logs.md
+  - pipelineruns.md: pipelineruns.md
+  - pipelines.md: pipelines.md
+  - resources.md: resources.md
+  - taskruns.md: taskruns.md
+  - tasks.md: tasks.md
+#- name: v0.9.1
+#  displayName: v0.8.x
+#  files:
+#  - README.md: _index.md
+#  - auth.md: auth.md
+#  - conditions.md: conditions.md
+#  - container-contract.md: container-contract.md
+#  - labels.md: labels.md
+#  - logs.md: logs.md
+#  - pipelineruns.md: pipelineruns.md
+#  - pipelines.md: pipelines.md
+#  - resources.md: resources.md
+#  - taskruns.md: taskruns.md
+#  - tasks.md: tasks.md
+#- name: v0.9.1
+#  displayName: v0.7.x
+#  files:
+#  - README.md: _index.md
+#  - auth.md: auth.md
+#  - conditions.md: conditions.md
+#  - container-contract.md: container-contract.md
+#  - labels.md: labels.md
+#  - logs.md: logs.md
+#  - pipelineruns.md: pipelineruns.md
+#  - pipelines.md: pipelines.md
+#  - resources.md: resources.md
+#  - taskruns.md: taskruns.md
+#  - tasks.md: tasks.md
```

Comment:
do we want to include the commented out sections?

Reply:
I included these commended out sections as an example for showcasing how to set up multiple document versions, maybe they are better presented as explanatory texts; I will update them.